File: week6/sentimental-mario-less/mario.py
from cs50 import get_int
# If not 1 to 8, reprompt
while True:
    height = get_int(("Height: "))
    if 0 < height < 9:
        break
# Print out staircase
i = 1
for i in range(height):
    print(" " * (height-(i+1)), end="")
    print("#" * (i+1))

File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Prompting for input
    int input;
    do
    {
        input = get_int("Height: ");
    }
    while (input < 1 || input > 8);

    //Print stairs
    {
        for (int loop = 0 ; loop < input ; loop++)
        {
            //Printing invsible backward steps
            for (int i = 0 ; i < input - (loop + 1) ; i++)
            {
                printf(" ");
            }
            //Printing visible LHS steps
            for (int i = 0 ; i < loop + 1 ; i++)
            {
                printf("#");
            }
            printf("\n");
        }
    }
}

